### PR TITLE
VWO Cloud Destination

### PR DESCRIPTION
**Pull request by prathameshwingify**
This is the implementation for VWO Cloud Destination. All tests pass locally.

_A summary of your pull request, including the what change you're making and why._

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [X] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [X] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/src/destinations/vwo/index.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  authentication: {
+    scheme: 'custom',
+    fields: {
+      vwoAccountId: {
+        label: 'VWO Account ID',
+        description: 'Enter your VWO Account ID',
+        type: 'number',
+        required: true
+      }
+    }
```
Comment (SyedWasiHaider) [inline packages/destination-actions/src/destinations/vwo/index.ts:28]: Do you have a way to verify that a `VWO Account ID` is valid? If so, I would recommend adding a `testAuthentication` method like we do in some of our other destinations:

https://github.com/segmentio/action-destinations/blob/a6b10d67a0b47f746ceee72b8293fd809dd3463b/packages/destination-actions/src/destinations/amplitude/index.ts#L101-L110

**Comment by SyedWasiHaider** [comment]
Looks mostly good to me. One small suggestion if you're able to but not blocking.

File: packages/destination-actions/src/destinations/vwo/index.ts (hunk 1 above)
Comment (prathameshwingify) [reply packages/destination-actions/src/destinations/vwo/index.ts:35]: Hi,
Currently, we don't have an API endpoint to verify the validity of an AccountID. However, I have added a basic range validation for now. I will update the same with a more robust mechanism when we come up with one.

**Review by dlasky — APPROVED** [review]

File: packages/destination-actions/src/destinations/vwo/trackEvent/__tests__/snapshot.test.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import { createTestEvent, createTestIntegration } from '@segment/actions-core'
+import { generateTestData } from '../../../../lib/test-data'
+import destination from '../../index'
+import nock from 'nock'
+
+const testDestination = createTestIntegration(destination)
+const actionSlug = 'trackEvent'
+const destinationSlug = 'vwo'
```
Comment (valerieernst) [inline packages/destination-actions/src/destinations/vwo/trackEvent/__tests__/snapshot.test.ts:8]: Can you update this to be the cloud destination slug `actions-vwo-cloud`?

File: packages/destination-actions/src/destinations/vwo/identifyUser/__tests__/snapshot.test.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import { createTestEvent, createTestIntegration } from '@segment/actions-core'
+import { generateTestData } from '../../../../lib/test-data'
+import destination from '../../index'
+import nock from 'nock'
+
+const testDestination = createTestIntegration(destination)
+const actionSlug = 'identifyUser'
+const destinationSlug = 'vwo'
```
[the remark this answers is not in the document]
Comment (prathameshwingify) [reply packages/destination-actions/src/destinations/vwo/identifyUser/__tests__/snapshot.test.ts]: It is updated

File: packages/destination-actions/src/destinations/vwo/trackEvent/__tests__/snapshot.test.ts (hunk 2 above)
Comment (prathameshwingify) [reply packages/destination-actions/src/destinations/vwo/trackEvent/__tests__/snapshot.test.ts]: It is updated

File: packages/destination-actions/src/destinations/vwo/pageVisit/__tests__/snapshot.test.ts [hunk 4]
```diff
@@ -0,0 +1,75 @@
+import { createTestEvent, createTestIntegration } from '@segment/actions-core'
+import { generateTestData } from '../../../../lib/test-data'
+import destination from '../../index'
+import nock from 'nock'
+
+const testDestination = createTestIntegration(destination)
+const actionSlug = 'pageVisit'
+const destinationSlug = 'Vwo'
```
[the remark this answers is not in the document]
Comment (prathameshwingify) [reply packages/destination-actions/src/destinations/vwo/pageVisit/__tests__/snapshot.test.ts]: It is updated

**Pull request merged**